src/main/java/io/micronaut/gradle/graalvm/NativeImageTask.java

```diff
@@ -105,6 +105,9 @@ public void configure() {
         String imageName = getImageName().get();
         args("-H:Name=" + imageName);
 
+        // to build a "mostly" static native image
+        args("-H:+StaticExecutableWithDynamicLibC");
```

Comment:
I think this argument should only be added automatically if the image is `distroless` since we can't assume this flag for all images

Reply:
Yes, you are right. I found an issue with previous changes, see https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/125#issuecomment-771758146. I'll wait for the user to fix his stuff and then I'll do my changes on top of his work.